src/lib/optimizer/chunk_statistics.cpp

```diff
@@ -0,0 +1,30 @@
+#include "chunk_statistics.hpp"
+
+#include <iterator>
+#include <sstream>
+
+#include "all_parameter_variant.hpp"
+#include "utils/assert.hpp"
+
+namespace opossum {
+
+bool ChunkStatistics::can_prune(const ColumnID column_id, const AllTypeVariant& value,
+                                const PredicateCondition scan_type) const {
+  DebugAssert(column_id < _statistics.size(), "the passed column id should fit in the bounds of the statistics");
+  return _statistics[column_id]->can_prune(value, scan_type);
+}
+
+std::string ChunkStatistics::to_string() const {
```

Comment:
I think we prefer (by usage vote ;) )  print(ostream&)-like methods over to_string() methods, but that's up to you